### PR TITLE
Improving tests

### PR DESCRIPTION
**Pull request by andrewgeorgiou1981**
The cookie-consent test was not testing the functionality correctly.

### CODE REVIEW

File: test/oPermutive.test.js [hunk 1]
```diff
@@ -124,16 +124,12 @@ describe("OPermutive", () => {
...
-				});
-			} catch(e) {
-				proclaim.equal(e instanceof Error, true);
-				proclaim.equal(e.message, 'o-permutive: Could not initialise. No consent found');
-			}
+			document.cookie = 'foo=foo;';
+			oPermutive.init({
+				projectId: "1",
+				publicApiKey: "key",
+				consentFtCookie: true
```
Comment (carlesandres) [inline test/oPermutive.test.js]: Doesn't `consentFTCookie: true` defeat the purpose of  `describe("Consent is NOT set",` ?

Comment (alevito) [reply test/oPermutive.test.js]: good point

Comment (andrewgeorgiou1981) [reply test/oPermutive.test.js]: no, but maybe it has caused some confusion in terms of the naming... as per the docs - setting consentFTcookie to true will mean that that o-permutive checks the consentCookie to derive the User's consent.

Comment (andrewgeorgiou1981) [reply test/oPermutive.test.js]: maybe this should be changed - to something like "consent : getConsentFromCookie()"

Comment (carlesandres) [reply test/oPermutive.test.js]: Yeah! It's a bit confusing

Comment (alevito) [reply test/oPermutive.test.js]: > maybe this should be changed - to something like "consent : getConsentFromCookie()"

Do you mean passing a callback as argument? Sounds like a good idea.
Alternatively you could rename the property `consentFtCookie` to `checkForConsent` or `checkForConsentFtCookie`?

Comment (andrewgeorgiou1981) [reply test/oPermutive.test.js]: > > maybe this should be changed - to something like "consent : getConsentFromCookie()"
> 
> Do you mean passing a callback as argument? Sounds like a good idea.
> Alternatively you could rename the property `consentFtCookie` to something like `checkForConsent` or `checkForConsentFtCookie`?

I like the second option as it also works well with passing config declaratively.

**Pull request merged**